packages/lab/src/Flow/types/flow.ts

```diff
@@ -8,57 +8,39 @@ import { HvColorAny } from "@hitachivantara/uikit-styles";
 
 // Node types
 
-type NodeExtras<GroupId extends keyof any = string, NodeData = any> = {
-  meta?: HvFlowNodeTypeMeta<GroupId, NodeData>;
-};
-
 /** HvFlowNode component type. @extends React.FC */
-export interface HvFlowNodeFC<
-  GroupId extends keyof any = string,
-  NodeData = any
-> extends FC<NodeProps<NodeData>>,
-    NodeExtras<GroupId, NodeData> {}
-
-export interface HvFlowNodeComponentClass<
-  GroupId extends keyof any = string,
-  NodeData = any
-> extends ComponentClass<NodeProps>,
-    NodeExtras<GroupId, NodeData> {}
-
-export type HvFlowNodeComponentType<
-  GroupId extends keyof any = string,
-  NodeData = any
-> =
-  | HvFlowNodeComponentClass<GroupId, NodeData>
-  | HvFlowNodeFC<GroupId, NodeData>;
-
-export type HvFlowNodeTypes<
-  GroupId extends keyof any = string,
-  NodeData = any
-> = Record<string, HvFlowNodeComponentType<GroupId, NodeData>>;
+export interface HvFlowNodeFC<NodeData = any> extends FC<NodeProps<NodeData>> {}
+
+export interface HvFlowNodeComponentClass extends ComponentClass<NodeProps> {}
+
+export type HvFlowNodeComponentType<NodeData = any> =
+  | HvFlowNodeComponentClass
+  | HvFlowNodeFC<NodeData>;
+
+export type HvFlowNodeTypes<NodeData = any> = Record<
+  string,
+  HvFlowNodeComponentType<NodeData>
+>;
+
+export type HvFlowGroupItem = {
+  type: string;
```

Comment:
It took me quite some type to understand that this was the `nodeType`. Should we change the name to `nodeType`?  Maybe it's only a matter of adding some docs or a me problem 😅